src/config/database.js

```diff
@@ -1,3 +1,6 @@
+// NOTE: for some reason the SignsFive API does
+// not like 'export default' being used here.
+// So, would like to dig in this sometime other time.
 module.exports = {
```

Comment:
I am not sure what is going on with this. I kept getting but that might be due to being used by sequelize before starting app. is that possible?

Reply:
@kratsg  ^^

Reply:
```
db/models/index.js
8:var config    = require(__dirname + '/../../config/database.js')[env];
14:  var sequelize = new Sequelize(config.database, config.username, config.password, config);
```

Reply:
So I guess figuring out how to grab the config here should be improved.

Reply:
Yeah, not high priority for one small file. 👍